.changeset/sixty-feet-try.md

```diff
@@ -0,0 +1,7 @@
+---
+"@localyze-pluto/components": patch
```

Comment:
according to [semantic versioning](https://semver.org/), a patch is a backward compatible bugfix.
I would say this is either a breaking change (because it changes the behavior of an existing component) or a minor change. And considering that semantic version explicitly says that a breaking change is a change with incompatible API changes, then I guess it should be a minor change. But I', not sure if we ever officially decided to follow that

Reply:
Changed to minor.